UnderWaterPhoto/Scenes/AuthScreen/AuthViewController.swift

```diff
@@ -18,35 +23,117 @@ final class AuthViewController: UIViewController {
     var presenter: AuthPresenterProtocol?
     
     // MARK: PrivateProperties
+    private var isExpandedLoginButton: Bool = false
+    private var passwordTextFieldCenterYConstraint: Constraint?
+    private var emailTextFieldCenterYConstraint: Constraint?
+    private var loginButtonCenterYConstraint: Constraint?
+    private var registrationButtonCenterYConstrain: Constraint?
+    
+    private lazy var logoLabel: UILabel = {
+        let label = UILabel()
+        label.backgroundColor = .blue
+        return label
+    }()
+    
+    private lazy var headTitle: UILabel = {
+        let label = UILabel()
+        label.text = "Создать аккаунт"
+        label.textAlignment = .center
+        label.backgroundColor = .clear
+        label.textColor = UIColor(named: "backgroundColorRegistrationButton")
+        return label
+    }()
+    
+    private lazy var textFieldStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = 16
+        stackView.alignment = .center
+        return stackView
+    }()
+    
+    private lazy var nameTextField: UITextField = {
+        let textField = CastomTextField()
+        textField.placeholder = "Имя".localized
+        return textField
+    }()
     
-    private lazy var loginButton: UIButton = {
-        let button = UIButton()
+    private lazy var emailTextField: UITextField = {
+        let textField = CastomTextField()
+        textField.placeholder = "E-mail".localized
+        return textField
+    }()
+    
+    private lazy var passwordTextField: UITextField = {
+        let textField = CastomTextField()
+        textField.placeholder = "Пароль".localized
+        textField.isSecureTextEntry = true
+        return textField
+    }()
+    
+    private lazy var repeatPasswordTextField: UITextField = {
+        let textField = CastomTextField()
+        textField.placeholder = "Повторите пароль".localized
+        textField.isSecureTextEntry = true
+        return textField
+    }()
+    
+    private lazy var loginButton: CastomButton = {
+        let button = CastomButton(frame: .zero)
+        button.type = .loginButton
         button.addTarget(self, action: #selector(loginButtonPressed), for: .touchUpInside)
+        button.setTitle("Войти", for: .normal)
+        button.contentHorizontalAlignment = .left
         return button
     }()
     
-    private lazy var registrationButton: UIButton = {
-        let button = UIButton()
+    private lazy var registrationButton: CastomButton = {
+        let button = CastomButton(frame: .zero)
+        button.type = .registrationButton
         button.addTarget(self, action: #selector(registrationButtonPressed), for: .touchUpInside)
+        button.setTitle("Зарегистрироваться", for: .normal)
```

Comment:
Давай здесь тоже сделаем .locazlized, если везде начали
```suggestion
        button.setTitle("Зарегистрироваться".localized, for: .normal)
```

Reply:
выполнил